README.md

```diff
@@ -13,3 +13,38 @@ A step-by-step description to run the tutorials with jupyter is located [here](d
 ## Limitations
 
 If you are using tobac 1.x, the notebooks here won#t work because of changes in the code structure. The equivalent notebooks can be found in the [examples](https://github.com/climate-processes/tobac/tree/master/examples) folder in the [tobac master branch](https://github.com/climate-processes/tobac/tree/master/examples).
+
+
+## Contributing
+You contribution is welcome! 
+
+Each additional tutorial notbeook helps new `tobac` users to understand the capability of the tracking software and supports creative use of feature identification and tracking in the atmospheric science context.
+
+
+### Pull Request Requirements
+
+Please provide a separate pull request:
+
+* **with your example notebook included**
+  * please extensively use markdown elements to structure and explain your notebook
+  * check already existing notebooks to get an idea about our tutorial style
+  * please upload your example data to [zenodo]( https://zenodo.org/ ) (see next paragraph)
+  * please check if your notebook is running in the recommended standard conda enviroment [see here]((docs/RunningTutorials.md) and update the [list of required python packages](./requirements.txt) if needed
+
+* **with your suggestions to change the documentation elements**, e.g. `index.rst`
+  * please care that sphinx-based rendering is still working correctly
```

Comment:
Maybe provide a link on how to check this or in general how jupyter notebook works with Sphinx (e.g. https://docs.readthedocs.io/en/stable/guides/jupyter.html) ?